templates/DatasetReview/dataset-links.html.twig

```diff
@@ -0,0 +1,140 @@
+<fieldset>
+    <legend>Dataset Links</legend>
+    <div id="dataset-links">
+        {% for link in form.datasetLinks %}
+        <fieldset>
+            <legend>Link: {{ loop.index }}</legend>
+            <table class="tab-table dataset-links" index="{{link.vars.name}}">
+            <tr>
+                <td>
+                    <span class="helpSection">
+                        <img class="info" src="{{ asset('images/info32.png') }}">
+                        <div class="tooltiptext">
+                            Please enter a ....
+                        </div>
+                    </span>
+                    <button class="deletebutton" type="button">Delete Link</button>
+                     {{ form_row(link.url) }}
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <span class="helpSection">
+                        <img class="info" src="{{ asset('images/info32.png') }}">
+                        <div class="tooltiptext">
+                            Please enter a ....
+                        </div>
+                    </span>
+                    {{ form_row(link.name) }}
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <span class="helpSection">
+                        <img class="info" src="{{ asset('images/info32.png') }}">
+                        <div class="tooltiptext">
+                            Please enter a ....
+                        </div>
+                    </span>
+                    {{ form_row(link.description) }}
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <span class="helpSection">
+                        <img class="info" src="{{ asset('images/info32.png') }}">
+                        <div class="tooltiptext">
+                            Please enter a ....
+                        </div>
+                    </span>
+                    {{ form_row(link.functionCode) }}
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <span class="helpSection">
+                        <img class="info" src="{{ asset('images/info32.png') }}">
+                        <div class="tooltiptext">
+                            Please enter a ....
+                        </div>
+                    </span>
+                    {{ form_row(link.protocol) }}
+                </td>
+            </tr>
+        </table>
+        </fieldset>
+        {% endfor %}
+    </div>
+
+    <button id="addLink" type="button">Add Link</button>
+</fieldset>
+
+<datalist id="dataLinkNames">
```

Comment:
Should we also add Dataland ? It is mentioned in the ticket
I think this should be a list of constants in the DatasetLink Entity